spec/models/fee_calculator_spec.rb

```diff
@@ -59,6 +59,50 @@
       it { is_expected.to eq(400) }
     end
 
+    context '12時間以内1200円' do
+      let(:attr) do
+        {
+         'type' => 'within',
+         'minute' => 720,
+         'fee' => 1200,
+         'repeat' => false,
+         'within' => {
+                      'type' => 'basic',
+                      'per_minute' => 60,
+                      'fee' => fee,
+                     }
```

Comment:
__[rubocop]__
* Indent the right brace the same as the start of the line where the left brace is.